tests/_utils/run_cdc_server

```diff
@@ -124,17 +132,23 @@ else
     $tls \
     $certcn \
     $addr \
-    $pd_addr &>> $workdir/stdout$log_suffix.log &
+    $pd_addr &>>$workdir/stdout$log_suffix.log &
 fi
 
 # Make sure cdc is ready to serve
-while ! cdc cli $pd_addr $tls changefeed list; do
-    i=$((i+1))
-    if [ "$i" -gt 50 ]; then
-        echo 'Failed to start TiCDC'
-        return 1
-    fi
-    sleep 3
+if [ -z "$tls_dir" ]; then
+  curl_status_cmd="curl -sL -w %{http_code}\n http://$addr_url/status -o /dev/null"
```

Comment:
Can this API ensure PD has necessary CDC metadata (eg, capture info) before it returning OK?

Reply:
I think so, because the ID of the info is returned here, and if that value is not set correctly, we won't get it. cc: @asddongmen What do you think?